cmd/git-chglog/initializer.go

```diff
@@ -30,7 +31,7 @@ func NewInitializer(
 	return &Initializer{
 		ctx:             ctx,
 		fs:              fs,
-		logger:          NewLogger(ctx.Stdout, ctx.Stderr, false, false),
+		logger:          NewLogger(colorable.NewColorableStdout(), colorable.NewColorableStderr(), false, false),
```

Comment:
I'd like to use `InitContext.Stdout` (and `.Sterr`) in `NewInitializer`, but do you have any special reasons?

Reply:
You are right. Reverted this part.